CHANGELOG.md

```diff
@@ -8,10 +8,18 @@ pull request if there was one.
 Current
 -------
 ### Added:
-- [CompositePhsyicalTable Core Components Refactor](https://github.com/yahoo/fili/pull/179)
+- [CompositePhysicalTable Core Components Refactor](https://github.com/yahoo/fili/pull/179)
     * Added `ConcretePhysicalTable` and `ConcreteAvailability` to model table in druid datasource and its availabillity in the new table availability structure
     * Added class variable for `DataSourceMetadataService` and `ConfigurationLoader` into `AbstractBinderFactory` for application to access
 
+- [PermissiveAvailability and PermissiveConcretePhysicalTable](https://github.com/yahoo/fili/pull/190)
+    * Added `PermissiveConcretePhysicalTable` and `PermissiveAvailability` to model table in druid datasource and its availability in the new table availability structure.
+    `PermissiveConcretePhysicalTable` and `PermissiveAvailability` extends `ConcretePhysicalTable` and `ConcreteAvailability`, respectively.
```

Comment:
It's OK to have too much detail in the changelog. It's easier to trim it out than to add it in, so it's better to err on the side of too much.

I tend to go back and clean up the changelogs when we go to cut the stable version anyways, so I wouldn't worry about this.